targets/kde

```diff
@@ -9,7 +9,7 @@ CHROOTBIN='crouton-noroot startkde'
 . "${TARGETSDIR:="$PWD"}/common"
 
 ### Append to prepare.sh:
-install --minimal kde-baseapps kde-runtime plasma-desktop pulseaudio \
+install --minimal kde-baseapps kde-runtime plasma-desktop pulseaudio kwin \
```

Comment:
kwin package doesn't exist on trusty (which isn't EOL'd quite yet), so please go through the various releases and make sure kwin exists on them.